rubrics/api/src/main/java/org/sakaiproject/rubrics/logic/model/Evaluation.java

```diff
@@ -58,7 +58,7 @@
         "criterionOutcomes", "metadata"})
 @NoArgsConstructor
 @Table(name = "rbc_evaluation",
-    uniqueConstraints = @UniqueConstraint(columnNames = { "association_id", "evaluated_item_id", "evaluator_id" })
+    uniqueConstraints = @UniqueConstraint(columnNames = { "association_id", "evaluated_item_id", "evaluator_id", "evaluated_item_owner_id" })
```

Comment:
This implies you can have multiple items evaluated by different owners which isn't what we would want.
Wondering if it makes sense to record the group id as the owner_id vs using the students id?

Reply:
That would break the student view, as it's directly checked on ddbb against the user's id. But yeah, this needs more thinking into it.

Reply:
What if we removed the evaluator_id from the key?